backbone.js

```diff
@@ -1408,6 +1408,31 @@
     return Backbone.$.ajax.apply(Backbone.$, arguments);
   };
 
+  // A psuedo Promise implementation used to ensure asynchronous methods
+  // return thenables.
+  // Override this if you'd like to use a different ES6 library.
+  Backbone.Promise = function() {
```

Comment:
I'd say a promise implementation should be included by default (I would suggest [`npo`](https://github.com/getify/native-promise-only))